src/Coveo.Connectors.EasyFilePusher/ProgramArguments.cs

```diff
@@ -0,0 +1,106 @@
+﻿using CommandLine;
+
+namespace Coveo.Connectors.EasyFilePusher
+{
+    /// <summary>
+    /// Definition of the program input arguments.
+    /// </summary>
+    public class ProgramArguments
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        [Option(Required = true, HelpText = "Cloud environment: " + nameof(CloudEnvironment.Hipaa) + ", " + nameof(CloudEnvironment.Prod) + ", " + nameof(CloudEnvironment.QA) + " or " + nameof(CloudEnvironment.Dev) + ".")]
+        public CloudEnvironment Environment { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Option(Required = true, HelpText = "Cloud region: " + nameof(CloudRegion.UsEast1) + " or " + nameof(CloudRegion.EuWest1) + ".")]
+        public CloudRegion Region { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Option(Required = true, HelpText = "ID of the organization in which to push documents.")]
+        public string Organization { get; set; } = "";
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Option(Required = true, HelpText = "ID of the source in which to push documents.")]
+        public string SourceId { get; set; } = "";
```

Comment:
I would either choose OrganizationId and SourceId or Organizaiton and Source

Reply:
Ton avis sur le sujet est sûrement plus éclairé que le-mien.  Entre "Source" et "SourceId", j'ai tendance à trouver "SourceId" plus clair.  Dans le sens que ça enlève la confusion possible que des utilisateurs pourraient autrement être tentés de penser qu'ils peuvent spécifier le nom de la source, alors qu'il faut que ça soit son ID.  Donc, pour suivre ta suggestion, j'aurais tendance à choisir ton premier choix, donc "OrganizationId" et "SourceId".  Ça t'irait?

Reply:
J'irais pour SourceId et OrganizationId aussi